test-cypress/cypress/commands/server.ts

```diff
@@ -0,0 +1,85 @@
+// load the global Cypress types
+/// <reference types="cypress" />
+export const DEFAULT_DATE = new Date(2021, 1, 16, 22);
+import url = require("url");
+
+export function apiPath() : string {
```

Comment:
Does this need to be an exported function?  Can it just be `export const API_PATH = Cypress.env('cacophony-api-server')` ?  I don't think it changes, and I assume that the env var is available at import time?